sentry-android-ndk/src/main/jni/sentry.c

```diff
@@ -212,3 +220,123 @@ Java_io_sentry_android_ndk_SentryNdk_initSentryNative(JNIEnv *env, jclass cls,
 
     sentry_init(options);
 }
+
+JNIEXPORT void JNICALL
+Java_io_sentry_android_ndk_NativeModuleListLoader_nativeClearModuleList(JNIEnv *env, jclass cls) {
+    sentry_clear_modulecache();
+}
+
+JNIEXPORT jobjectArray JNICALL
+Java_io_sentry_android_ndk_NativeModuleListLoader_nativeLoadModuleList(JNIEnv *env, jclass cls) {
+    sentry_value_t image_list_t = sentry_get_modules_list();
+    jobjectArray image_list = NULL;
+
+    if (sentry_value_get_type(image_list_t) == SENTRY_VALUE_TYPE_LIST) {
+        size_t len_t = sentry_value_get_length(image_list_t);
+
+        jclass image_class = (*env)->FindClass(env, "io/sentry/protocol/DebugImage");
```

Comment:
Do we expect this method to be called multiple times? I wonder if would be worth caching these FindClass results

Reply:
unless somebody call `clearDebugImages` its used only once